README.md

```diff
@@ -98,6 +98,24 @@ $ uname -a | curl http://localhost:8080/function/nodejs-echo--data-binary @-
 
 > For further instructions on the manual CLI flags (without using a YAML file) read [manual_cli.md](https://github.com/alexellis/faas-cli/blob/master/MANUAL_CLI.md)
 
+#### Adding custom language support
```

Comment:
Not just custom language, this could be a pre-existing binary. For instance take the samples here -> https://github.com/alexellis/faas/tree/master/sample-functions

and here -> https://github.com/alexellis/faas/blob/master/docker-compose.yml